### PR TITLE
Added 'Draw picture' and 'post question online' to korean

### PR DESCRIPTION
**Pull request by BayMinimum**
Please review this, @casey-collab ! XD

### CODE REVIEW

File: korean/ask-kin/ask-add-100.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+여태까지 초딩들에게 답변을 해 주면서 모은 내공을 탏탏 털어넣었다...
```
Comment (casey-collab) [inline korean/ask-kin/ask-add-100.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: korean/ask-kin/ask-kin.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+글을 등록했다!
```
Comment (casey-collab) [inline korean/ask-kin/ask-kin.md]: This file has also been added to a directory that already contained a page.

File: korean/draw-picture/chocopen.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+우왕 쓱삭쓱삭 잘 그려지네요!
```
Comment (casey-collab) [inline korean/draw-picture/chocopen.md]: This file has also been added to a directory that already contained a page.

File: korean/draw-picture/eat-choco.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+너무 달아요..!
```
Comment (casey-collab) [inline korean/draw-picture/eat-choco.md]: This file has also been added to a directory that already contained a page.

File: korean/draw-picture/woodstick.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+나무 꼬치로 선을 긋기 위에 마시멜로 벽을 쿡 찔러보았어요.
```
Comment (casey-collab) [inline korean/draw-picture/woodstick.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by BayMinimum** [comment]
Thanks! I have commited fixes for the things above on my branch... Can you review this again?

**Comment by BayMinimum** [comment]
@casey-collab

**Pull request closed without merging**